src/core/dev/net_device_table_mgr.cpp

```diff
@@ -245,7 +245,9 @@ void net_device_table_mgr::update_tbl()
                                                sa_family_t family) {
                     auto it = net_device_map.find(ip->local_addr);
                     if (it != net_device_map.end()) {
-                        duplicate_ips.insert(ip->local_addr.to_str(family));
+                        if (!(family == AF_INET6 && ip->local_addr.is_linklocal(family))) {
```

Comment:
please put a comment with motivation for this exception. Mention that logical interfaces can have duplicate IPv6 link-local and the warning is not relevant because (1) user needs to specify interface along with link-local address and (2) XLIO doesn't support IPv6 link-local at the moment